test/compile/data.test.ts

```diff
@@ -342,6 +343,7 @@ describe('data: timeUnitDomain', function() {
     });
 
     it('should be compiled into correct string set', function() {
+      model.component.data = {} as DataComponent;
```

Comment:
Having to do this for every line of test looks annoying.   I think it doesn't hurt to init `data: {}` above.